### PR TITLE
Remove everything related to the old deposit return voucher logic (Apps-1644)

### PR DESCRIPTION
**Pull request by Fabtron**
The old logic related to the deposit return voucher has been removed. 

APPS-1644

### How to test?
Launch the app and test shopping cart related stuff. Scan or add different products to see if the product look up still works.

### Definition of Done

- [x] Issue is linked
- [x] All requirements of the issue are fulfilled
- [x] Changelog is updated
- [x] Documentation is updated
- [x] [Self-Review](https://www.nerdwallet.com/blog/engineering/why-you-should-be-doing-self-reviews/)
- [ ] Review with the Product Owner _(Release-Variante o. Minified Build)_

#### App Tests
- [x] Minified build has been tested _(aka. Release Build)_
- [ ] Environments have been taken care of _(Production/Staging)_
- [ ] Supported languages have been tested
- [ ] Light-/Dark-Mode has been tested
- [ ] Edge-Cases have been tested
- [ ] Android API Levels have been taken care of _(minSdk?)_

#### Testing
- [ ] Tests have been written _(aka Unit-Tests, Integration-Tests, ...)_

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@ All notable changes to this project will be documented in this file.
 ## UNRELEASED
 ### Added
 ### Changed
-### Removed
+### Removed/
```
Comment (cmaier) [inline CHANGELOG.md:7]: Typo

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ All notable changes to this project will be documented in this file.
...
+* ui / core : Remove everything related to the old deposit return voucher feature
```
Comment (cmaier) [inline CHANGELOG.md:8]: Typo, whispace before colon.
Also, in the past, we've written several module names w/o spaces.

**Review by cmaier — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Fabtron) [reply CHANGELOG.md]: changed

File: CHANGELOG.md (hunk 2 above)
Comment (Fabtron) [reply CHANGELOG.md]: changed

**Review by cmaier — APPROVED** [review]
